### ISSUE TITLE
simpler type definition?

### ISSUE THREAD

**Issue by gregveres** [frenic/csstype#83]
I just found your project and I think it is fantastic!

Would you consider adding a type defintion like this:

type Styles = Properties<string | number>;

and then export it? 

I find the notation of:

let style: CSS.Properties<string | number> = ...
to be a bit cumbersome when it could be something like:

let style: CSS.Style = ...

I know there are the other Property types (I did read the docs), but I suspect that 80% of the use cases fall under the above property use and those all could be simplified. For now I am going to do something like this in my code, but it would be great if it could be rolled into the main repo.

**Comment by meyer** [comment]
@gregveres for my personal projects where I use csstype, I define the type in a shared location and then use my own definition:

`types.ts`
```ts
export type CSSProperties = Properties<string | number>;
```

`somewhere-else.ts`
```ts
import { CSSProperties } from '~/types';

interface ExampleProps extends Pick<CSSProperties, 'color' | 'backgroundColor'> {
  // ...
}

const ExampleComponent: React.FC<ExampleProps> = ({ color, backgroundColor }) => {
  // ...
}
```

**Comment by strdr4605** [comment]
Just do the fix in your own project:
```ts
// extendCsstype.ts
// import original module declarations
import * as CSS from "csstype";

// and extend them!
declare module "csstype" {
  export type Styles = CSS.Properties<string | number>;
}
```

```ts
// in another file
import { Styles } from "csstype";

const style: Styles = {
  ...
}
```

**Comment by gregveres** [comment]
@strdr4605 Yes I did that (or something similar) but I think it would be better if people using the package didn't have to do that and that there was a standard, consistent way. Why should everybody have to re-invent the wheel?
